src/lib/style-properties.html

```diff
@@ -256,15 +307,55 @@
           hostSelector;
         var hostRx = new RegExp(this.rx.HOST_PREFIX + rxHostSelector +
           this.rx.HOST_SUFFIX);
+
+        var keyframesRules = element._styles._keyframes;
```

Comment:
Seems like this keyframes specific handling should be factored into a separate function